x-pack/platform/plugins/shared/fleet/server/services/epm/packages/cache.ts

```diff
@@ -52,6 +57,25 @@ class CacheSession {
     }
     return this._agentTemplateAssetsMap;
   }
+
+  getHandlebarsCompiledTemplateCache() {
```

Comment:
worth adding some test for these changes in `cache.test.ts`?

Reply:
Added in https://github.com/elastic/kibana/pull/213587/commits/a75245719e1d2d3543d353a7719a030b4aec7563